report/how_to.md

```diff
@@ -0,0 +1,70 @@
+# User Guide
+
+## How to use this book
+
+### Validation
+
+TODO
+
+### Verification
+
+Each verification case was created using either the [`MES`](./verification/methods.md/#method-of-exact-solutions-mes) or [`MMS`](./verification/methods.md/#method-of-manufactured-solutions-mms) and are tagged accordingly.
+
+TODO
```

Comment:
I don't think we need this in How to use this book since it's already explained in the intro.